.github/workflows/build-and-deploy-production.yml

```diff
@@ -60,8 +60,6 @@ jobs:
           HEAPIO_ID=${{ env.HEAPIO_ID }}
           FRONTEND_ORIGIN=${{ env.FRONTEND_ORIGIN }}
         push: true
-        cache-from: type=gha
-        cache-to: type=gha,mode=max
```

Comment:
Why remove cache, if you change namespace?

Reply:
Because there is nothing to cache now anyways and im [ rebuilding the cache flow](https://github.com/sidestream-tech/hanselmann-os/pull/4054#issuecomment-2604991137) for our projects; so it would be outdated anyways. On top i just want to ensure that nothing unforeseen can happen right now due to my unavailability for the next 2 weeks.